QuickBooks/quickbooks.yaml

```diff
@@ -606,8 +632,12 @@ components:
           type: string
           description: 'Shipping provider''s tracking number for the delivery of the goods associated with the transaction.'
         PrintStatus:
-          type: string
+          type: select
```

Comment:
This doesn't seem to be valid type.
Please use enum:
https://swagger.io/docs/specification/data-models/enums/